packages/coinstac-ui/app/render/components/consortium/consortium-form.js

```diff
@@ -76,102 +75,52 @@ class ConsortiumForm extends Component {
       errors.label = 'Name required';
     }
 
+    /**
+     * Find the `covariates` computation input and verify every covariate has a
+     * `name` and `type` property.
+     *
+     * @todo Validate is a static method and doesn't have access to the
+     * `computationInputs` property. Improve duck-like `covariateIndex` checks!
+     */
+    const covariatesIndex = values.activeComputationInputs.findIndex(input => (
+      Array.isArray(input) &&
+      input.length &&
+      typeof input[0] === 'object'
+    ));
+
+    if (
+      covariatesIndex > -1 &&
+      values.activeComputationInputs[covariatesIndex].some((covariate) => !(
+        covariate.name.length &&
```

Comment:
These checks become a little gross, but Redux Form doesn't seem to like passing configuration with props. `values` contains only form input.